src/explorer-components/BountyPayoutData/BountyPayoutData.module.scss

```diff
@@ -0,0 +1,10 @@
+.payoutEthBox {
+    margin-top: 0.5rem;
```

Comment:
Replace margins with corresponding variables (i.e. `$m-spacing`). Check `styles/variables.scss` for a full list of vars